### DISCUSSION TITLE
［灌水］最近看大家都在玩黑神话悟空 然后我也买了一份不知道是不是正版

### DISCUSSION THREAD

**Discussion by abyssal-sea-c** [share121/inter-knot#5458] [灌水]
看样子应该很好玩吧
![1724175541933](https://github.com/user-attachments/assets/f518e794-72e8-4f96-9b5e-30e7eb99331b)

**Comment by swyh0648** [comment]
![wx_camera_1726648603932](https://github.com/user-attachments/assets/27682e3a-bf76-4300-aa1d-7dc8d44c0026)
也就这样
